tests/contest/contest/src/tests/cgroups/cpu/v2.rs

```diff
@@ -95,23 +95,20 @@ fn test_cpu_weight_zero_ignored() -> TestResult {
     })
 }
 
-/// Tests if a cpu weight that is too high (over 10000 after conversion) is set to the maximum value
+/// Tests behavior when setting a cpu weight that is too high
 fn test_cpu_weight_too_high_maximum_set() -> TestResult {
     let cpu_weight = 500_000u64;
-    let converted_cpu_weight = 10_000;
     let cpu = test_result!(LinuxCpuBuilder::default()
         .shares(cpu_weight)
         .build()
         .context("build cpu spec"));
-
     let spec = test_result!(create_spec("test_cpu_weight_too_high_maximum_set", cpu));
-    test_outside_container(&spec, &|data| {
-        test_result!(check_container_created(&data));
-        test_result!(check_cpu_weight(
-            "test_cpu_weight_too_high_maximum_set",
-            converted_cpu_weight
-        ));
-        TestResult::Passed
+    test_outside_container(&spec, &|data| match check_container_created(&data) {
```

Comment:
What changed?

Reply:
The test was updated to handle two valid behaviors across different version OS.

Before: silently cap weight to 10000 and succeed
After: reject out-of-range values and fail container creation

Since both behaviors are valid according to kernel documentation (which specifies the range as [1, 10000]), the test now passes in either case, making it compatible with both implementations.

I added a comment because it was unclear.